tests/Integration/all_integration_server_tests.sh

```diff
@@ -65,6 +65,12 @@ python $SERVERINSTALLDIR/DIRAC/tests/Integration/DataManagementSystem/Test_Clien
 diracDFCDB 2>&1 | tee -a $SERVER_TEST_OUTPUT
 python $SERVERINSTALLDIR/DIRAC/tests/Integration/DataManagementSystem/Test_FileCatalogDB.py 2>&1 | tee -a $SERVER_TEST_OUTPUT; (( ERR |= $? ))
 
+
+#-------------------------------------------------------------------------------#
+echo -e "*** $(date -u)  **** FTS TESTS ****\n"
+# I know, it says Client, but it also instaciates a DB, so it needs to be here
```

Comment:
You might consider splitting in 2 tests, if it's not too difficult.

Reply:
I considered, but it actually is difficult.  I could write a new Client which just does put and get for a client, but that's a bit trivial. That's why I kept them mixed